test/e2e/gardener/shoot/create_update_delete.go

```diff
@@ -41,17 +41,15 @@ var _ = Describe("Shoot Tests", Label("Shoot", "default"), func() {
 
 	// explicitly use one version below the latest supported minor version so that Kubernetes version update test can be
 	// performed
-	// TODO(ialidzhikov): Update to 1.26.0 after the merge of https://github.com/gardener/gardener/pull/7275.
-	f.Shoot.Spec.Kubernetes.Version = "1.25.4"
+	f.Shoot.Spec.Kubernetes.Version = "1.26.0"
```

Comment:
From the comment for this line,
```
// explicitly use one version below the latest supported minor version so that Kubernetes version update test can be
// performed
```
Should the Kubernetes version for the shoot not be 1.25.x?
/cc @rfranzke

Reply:
Yes, you are right. It seems that this version should have stay 1.24.8 in https://github.com/gardener/gardener/pull/7275 but I bumped it to 1.25.4. 😞 We don't need version update in this place.